swift/extractor/main.cpp

```diff
@@ -54,12 +59,26 @@ int main(int argc, char** argv) {
   configuration.scratchDir = getenv_or("CODEQL_EXTRACTOR_SWIFT_SCRATCH_DIR", ".");
 
   configuration.tempTrapDir = configuration.scratchDir + "/swift-trap-temp";
+  configuration.VFSDir = configuration.scratchDir + "/swift-vfs";
+  configuration.tempVFSDir = configuration.scratchDir + "/swift-vfs-temp";
+  configuration.tempArtifactDir = configuration.scratchDir + "/swift-extraction-artifacts";
```

Comment:
the scratch dir is already swift-extractor specific, so I don't think we need the `swift-` prefixes here.

Reply:
ah, I thought `CODEQL_EXTRACTOR_SWIFT_SCRATCH_DIR` meant it was extractor specific? But ok.